app/src/main/java/ch/epfl/sdp/drone3d/service/impl/drone/DroneExecutorImpl.kt

```diff
@@ -47,7 +48,9 @@ class DroneExecutorImpl(
             DroneUtils.generateMissionItem(
                 home.latitudeDeg,
                 home.longitudeDeg,
-                home.relativeAltitudeM))
+                home.relativeAltitudeM,
+                missionPlan.missionItems[0].gimbalPitchDeg,
```

Comment:
This is the last point of the mission, just above the launch point.

I don't think we need to specify an angle. And the angle should probably not be the one of the first mission item.
The altitude of this point should be changed to the one of the mission once it is implemented